Lib/test/_test_multiprocessing.py

```diff
@@ -2571,6 +2571,15 @@ def raising():
 def unpickleable_result():
     return lambda: 42
 
+def waiting(args):
+    time.sleep(7)
```

Comment:
I suppose it's not gonna wait that long when running the test? Just checking.

Reply:
Revised tests. I wanted to make sure we did not break the pool before any tasks were running. Now I've reduced that wait to 0.5 sec - https://github.com/python/cpython/pull/10441/files#diff-b046ab474480855fb4a01de88cfc82bbR2672